### ISSUE TITLE
[Accessibility]: Difficult to navigate to code for blob view

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#35072]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

When navigating a page in Sourcegraph, there are a very large number of elements that need to be tabbed through to reach the code blob view.

### Expected behavior

It should be possible to quickly navigate to the code view, since that is one of the main parts of the page.

### Additional details

Potential fix: Add header for code view. This should allow fast navigation by using Control + Option + Command + (Shift) + H.

Note that other large parts of the UI are also lacking headers -- perhaps there could be headers for the products layer, the search bar layer, the navigation and metadata layer (the filepath breadcrumbs + brain dot, github link, etc), the sidebar, and the reference panel (if one is present). However, that's not needed as part of this ticket.

https://user-images.githubusercontent.com/93103176/167205034-398297d0-c09e-438d-98a2-3179e3a2e262.mov

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [x] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [x] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Might be fixed by https://github.com/sourcegraph/sourcegraph/issues/36581
